packages/identity/src/buffer.ts

```diff
@@ -13,3 +13,23 @@ export function fromHexString(hexString: string): ArrayBuffer {
 export function toHexString(bytes: ArrayBuffer): string {
   return new Uint8Array(bytes).reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
 }
+
+/**
+ * Compares two ArrayBuffers and returns 0 if they are identical
+ * @param b1 {ArrayBuffer} Buffer 1
+ * @param b2 {ArrayBuffer} Buffer 2
+ */
+export function compare(b1: ArrayBuffer, b2: ArrayBuffer): number {
```

Comment:
I don't see this being used anywhere in this PR. Is it needed? If yes, would it make sense to add unit tests for this?

Reply:
Yep, seems we didn't need it